src/connectionController.ts

```diff
@@ -69,21 +72,30 @@ interface ConnectionQuickPicks {
   data: { type: NewConnectionType; connectionId?: string };
 }
 
-interface ConnectionSecretsInfo {
-  connectionId: string;
-  secrets: ConnectionSecrets;
-}
+type StoreConnectionInfoWithConnectionModel = StoreConnectionInfo &
+  Required<Pick<StoreConnectionInfo, 'connectionModel'>>;
 
 type StoreConnectionInfoWithConnectionOptions = StoreConnectionInfo &
   Required<Pick<StoreConnectionInfo, 'connectionOptions'>>;
 
+type MigratedStoreConnectionInfo = StoreConnectionInfo &
```

Comment:
Maybe `StoreConnectionInfoWithSecretStorageLocation` would match closer to other used variables.

Reply:
Yea it definitely would. Thank you :)

Reply:
Addressed this in the [following commit](https://github.com/mongodb-js/vscode/pull/552/commits/808e60e8b6d1b60ee5d71103c2ff858bb8baaa7d)